src/System.IO.Packaging/src/project.json

```diff
@@ -1,14 +1,20 @@
 {
-  "dependencies": {
-    "System.Collections": "4.0.10",
-    "System.Diagnostics.Debug": "4.0.10",
-    "System.IO.Compression": "4.0.0",
-    "System.IO.FileSystem": "4.0.0",
-    "System.IO.FileSystem.Primitives": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Xml.ReaderWriter": "4.0.0"
-  },
   "frameworks": {
-    "dnxcore50": {}
+    "dnxcore50": {
```

Comment:
New convention. Can you please change dnxcore50 to dotnet5.4 for this project to match the default PackageTargetFramework